### PR TITLE
Adds instructions on using future flag

### PR DESCRIPTION
**Pull request by elfisher**
Adds instructions for using future flag in README when testing blogs published in future

Addresses issue #280

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ If you are a partner, you are welcome to add your logo/link to our partners page
 This site uses [Jekyll](https://jekyllrb.com/). You can build the site and make it available on a local server via `docker-compose up -d`, or by installing all the dependencies on your local environment as follows.
 
 1. Install [Ruby](https://www.ruby-lang.org/en/) and [Bundler](https://bundler.io/), then run `bundle install`.
-2. Build and start Jekyll with `bundle exec jekyll serve`.
+2. Build and start Jekyll with `bundle exec jekyll serve`. **NOTE** To test blogs with a future publication date use `bundle exec jekyll serve --future`
```
Comment (stockholmux) [inline README.md:38]: As per my comment on #280 can we be less prescriptive here and link to the Jekyll docs? Teach a man to fish and all that.

I'd rather keep the number instructions clean, so I'd be in favour of putting this below the number instructions and without the all caps/bold "NOTE."

File: README.md [hunk 2]
```diff
@@ -78,4 +78,4 @@ This website was forked from the BSD-licensed [djangoproject.com](https://github
 
 ## Copyright
 
-Copyright OpenSearch Contributors. 
\ No newline at end of file
+Copyright OpenSearch Contributors.
```
Comment (stockholmux) [inline README.md:81]: Is this some sort of weird linting?

File: README.md (hunk 1 above)
Comment (stockholmux) [reply README.md:38]: https://jekyllrb.com/docs/configuration/options/

**Review by stockholmux — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (elfisher) [reply README.md]: adjusted accordingly

File: README.md (hunk 2 above)
Comment (elfisher) [reply README.md:81]: I believe so. It appears to have removed and re-add this line.

**Comment by elfisher** [comment]
@stockholmux updated based on your feedback.

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**